mod/amcd.php

```diff
@@ -1,8 +1,9 @@
 <?php
 
+use Friendica\App;
+
 function amcd_content(App $a) {
-//header("Content-type: text/json");
-echo <<< EOT
+	echo <<< JSON
```

Comment:
Yes, in the HereDoc syntax you can use whatever delimiter you want. I'm used to put the expected format of the following string for better expectations.